### DISCUSSION TITLE
Semillitas

### DISCUSSION THREAD

**Discussion by ELProductions** [iptv-org/community#765] [Q&A]
### Channel Name

Semillitas

### Channel ID

SemillitasTV.us

### Website

https://semillitas.com/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19762
